src/cross_lib/sound/cc65/vic20/vic20_sounds.c

```diff
@@ -52,7 +52,7 @@
 		VIC.volume_color |= 0x08; 

```

Comment:
I don't think that the optimizer will change that one.  Try changing it to
```c
VIC.volume_color = 0x08;
```
and, see if the code size is reduced.